index.html

```diff
@@ -18,6 +18,10 @@
 <body>
   <div class="ellipse-one"></div>
   <div class="ellipse-two"></div>
+  <div class="ellipse-three"></div>
+  <div class="ellipse-four"></div>
+  <div class="ellipse-five"></div>
+  <div class="ellipse-six"></div>
```

Comment:
- [ ] [OPTIONAL] You should consider moving the ellipsis icons as on a full HD screen (1920x1080) the website renders like this, with the ellipsis behind the text, which makes it unreadable and not accessible at all.

![](https://i.imgur.com/sKUGy16.png)